docs/DeveloperGuide.adoc

```diff
@@ -297,86 +299,263 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new problem |keep track of the problems for future usage
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a problem |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a problem by keyword |locate details of problems without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user |do advanced search on problems |locate details of problems without having to go through the entire list
```

Comment:
Maybe you could specify what 'advanced search' entails? Some more details would be appreciated.